src/main/java/org/broadinstitute/dsm/careevolve/Order.java

```diff
@@ -0,0 +1,43 @@
+package org.broadinstitute.dsm.careevolve;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import com.google.gson.annotations.SerializedName;
+
+public class Order {
+
+    @SerializedName("AOEs")
+    private List<AOE> aoes = new ArrayList<>();
+
+    @SerializedName("collection")
```

Comment:
Do these need to be capitalized as well? (`Collection`, `Patient`, etc). Also, do we need to hardcode things like `TestCode`?

Reply:
Yep!  Thanks for pointing that out.